api/doc/openapi.yaml

```diff
@@ -488,6 +490,22 @@ components:
       example: 25
       schema:
         type: integer
+    StartEvent:
+      name: start
+      in: query
+      required: false
+      description: The start index of events emitted from the account
```

Comment:
The start sequence number in the EVENT STREAM, defaulting to the latest event. The events are returned in the reverse order of sequence numbers.

Reply:
updated the description and remove the default 0 for start